scene/gui/base_button.cpp

```diff
@@ -341,6 +341,10 @@ void BaseButton::set_pressed(bool p_pressed) {
 	if (p_pressed) {
 		_unpress_group();
 	}
+	if (get_script_instance()) {
+		get_script_instance()->call(SceneStringNames::get_singleton()->_toggled, status.pressed);
+	}
+	emit_signal("toggled", status.pressed);
```

Comment:
The signal will be emitted twice in this situation:
https://github.com/godotengine/godot/blob/2b21bfa8c77d304da7cd28896c0aa08c1b3436d9/scene/gui/base_button.cpp#L483-L486

There's a lot of code duplication in `BaseButton` regarding emitting "pressed", "toggled", notifying the change, etc. Might be worth refactoring this a bit to make sure the behaviour is always consistent.

Reply:
yes, right... I will try. it became a big issue suddenly... :smile: